nuxeo-connect-client/src/main/java/org/nuxeo/connect/packages/dependencies/TargetPlatformFilterHelper.java

```diff
@@ -20,23 +20,46 @@
 import org.apache.commons.io.FilenameUtils;
 import org.apache.commons.io.IOCase;
 import org.apache.commons.lang3.StringUtils;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.nuxeo.connect.packages.dependencies.versioning.PlatformVersion;
+import org.nuxeo.connect.packages.dependencies.versioning.PlatformVersionRange;
 import org.nuxeo.connect.update.Package;
 
 /**
  * @author <a href="mailto:td@nuxeo.com">Thierry Delprat</a>
  */
 public class TargetPlatformFilterHelper {
 
-    public static boolean isCompatibleWithTargetPlatform(Package pkg, String targetPlatform) {
-        return isCompatibleWithTargetPlatform(pkg.getTargetPlatforms(), targetPlatform);
+    protected static Log log = LogFactory.getLog(TargetPlatformFilterHelper.class);
+
+    public static boolean isCompatibleWithTargetPlatform(Package pkg, String targetPlatform,
+            String targetPlatformVersion) {
+        String tpRangeSpec = pkg.getTargetPlatformRange();
```

Comment:
You don't need this variable before the `try/catch`

Reply:
It was to do the get only once, but I should use it in the if